elastic/integrations
# Cisco FTD | Create `security_event` fields from flattened `security` field

## Proposed commit message
- WHAT: Parses out most fields from flattened `cisco.ftd.security` into new field `cisco.ftd.security_event`.
- WHY: Allows aggregation of fields under `cisco.ftd.security_event` which was not possible due to flattened `cisco.ftd.security` field.



## Checklist


## How to test this PR locally


Run this command for pipeline tests: 
`$ elastic-package stack down && elastic-package build && elastic-package stack up --version=8.10.3 -d -v && eval "$(elastic-package stack shellinit)" && elastic-package test pipeline --generate -v`

Expected files are now updated with new field `cisco.ftd.security_event` containing the fields listed from the linked issue.
```
Run pipeline tests for the package
2023/11/14 11:36:16 DEBUG GET https://127.0.0.1:5601/api/status
--- Test results for package: cisco_ftd - START ---
╭───────────┬─────────────┬───────────┬────────────────────────────────┬────────┬──────────────╮
│ PACKAGE   │ DATA STREAM │ TEST TYPE │ TEST NAME                      │ RESULT │ TIME ELAPSED │
├───────────┼─────────────┼───────────┼────────────────────────────────┼────────┼──────────────┤
│ cisco_ftd │ log         │ pipeline  │ test-asa-fix.log               │ PASS   │       5.95ms │
│ cisco_ftd │ log         │ pipeline  │ test-asa.log                   │ PASS   │ 141.360791ms │
│ cisco_ftd │ log         │ pipeline  │ test-dns.log                   │ PASS   │  28.012791ms │
│ cisco_ftd │ log         │ pipeline  │ test-filtered.log              │ PASS   │   3.302541ms │
│ cisco_ftd │ log         │ pipeline  │ test-firepower-management.log  │ PASS   │  31.007625ms │
│ cisco_ftd │ log         │ pipeline  │ test-ftd-endpoint-profile.log  │ PASS   │  49.369333ms │
│ cisco_ftd │ log         │ pipeline  │ test-ftd-fix.log               │ PASS   │      7.219ms │
│ cisco_ftd │ log         │ pipeline  │ test-ftd-inbound-outbound.log  │ PASS   │  20.108375ms │
│ cisco_ftd │ log         │ pipeline  │ test-intrusion.log             │ PASS   │  18.601208ms │
│ cisco_ftd │ log         │ pipeline  │ test-no-type-id.log            │ PASS   │  14.540875ms │
│ cisco_ftd │ log         │ pipeline  │ test-not-ip.log                │ PASS   │    4.35875ms │
│ cisco_ftd │ log         │ pipeline  │ test-sample.log                │ PASS   │  30.343333ms │
│ cisco_ftd │ log         │ pipeline  │ test-security-connection.log   │ PASS   │    10.5255ms │
│ cisco_ftd │ log         │ pipeline  │ test-security-file-malware.log │ PASS   │  32.704292ms │
│ cisco_ftd │ log         │ pipeline  │ test-security-malware-site.log │ PASS   │   7.476917ms │
╰───────────┴─────────────┴───────────┴────────────────────────────────┴────────┴──────────────╯
--- Test results for package: cisco_ftd - END   ---
Done
```

## Related issues


- Closes https://github.com/elastic/integrations/issues/5769
